helpers/api/bounty/read-closed.ts

```diff
@@ -111,6 +111,13 @@ export default async function readBountyClosed(events, network: Network_v2, cust
             bounty.merged = proposal.scMergeId;
             bounty.state = "closed";
             await bounty.save();
+            await Promise.all(networkBounty?.proposals?.[0].details.map(async(detail) =>
+              await models.userPayments.create({
```

Comment:
you don't need to await a promise inside the Promise.all mapper, as `Promise.all` accepts an array of pending promises